src/main/java/org/example/CalendarManagement/calendarfacade/MeetingFacade.java

```diff
@@ -0,0 +1,121 @@
+package org.example.CalendarManagement.calendarfacade;
+
+import org.example.CalendarManagement.api.Response;
+import org.example.CalendarManagement.api.request.AddMeetingDataRequest;
+import org.example.CalendarManagement.calendarpersistence.model.MeetingRoom;
+import org.example.CalendarManagement.calendarpersistence.repository.EmployeeRepository;
+import org.example.CalendarManagement.calendarpersistence.repository.MeetingRoomRepository;
+import org.example.CalendarManagement.calendarservice.implementation.MeetingRoomService;
+import org.example.CalendarManagement.thriftclients.interfaces.MeetingServiceClient;
+import org.example.CalendarThriftConfiguration.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+public class MeetingFacade {
+    @Autowired
+    MeetingRoomService meetingRoomService;
+    @Autowired
+    MeetingServiceClient meetingServiceClient;
+
+    @Autowired
+    MeetingRoomRepository meetingRoomRepository;
+
+    @Autowired
+    EmployeeRepository employeeRepository;
+
+    @Transactional
+    public Response scheduleMeeting(AddMeetingDataRequest addMeetingDataRequest){
+        Response response =null;
+        Date dateOfMeeting = new Date(addMeetingDataRequest.getDateOfMeeting().getDayOfMonth(),addMeetingDataRequest.getDateOfMeeting().getMonthValue(),addMeetingDataRequest.getDateOfMeeting().getYear());
```

Comment:
avoid using java util Date

Reply:
This is custom class date not java util date